docs/becoming-a-maintainer/translations/es/como-configurar-tu-proyecto.md

```diff
@@ -0,0 +1,208 @@
+# Cómo configurar tu proyecto Open Source
+
+Hay muchas cosas a considerar al configurar tu proyecto Open Source. En esta parte del curso, revisaremos los componentes clave que todo proyecto necesita para tener éxito.
+
+## README detallado
+
+Un archivo README contiene una introducción a tu proyecto Open Source. Un buen README debe ser claro, conciso, actualizado y detallado. Este archivo se encuentra en tu directorio raíz y sirve como la primera impresión para tus colaboradores.
```

Comment:
Consider changing "Un archivo README contiene una introducción a tu proyecto Open Source." 

to: "Un archivo README contiene una introducción a tu proyecto **de** Open Source."